sale_stock_medical_prescription_us/models/medical_prescription_order_line.py

```diff
@@ -44,53 +43,65 @@ class MedicalPrescriptionOrderLine(models.Model):
     @api.multi
     def _compute_dispense_remain(self):
         day_uom_id = self.env.ref('medical_medication.product_uom_day')
-        for rec_id in self:
-            if not rec_id.duration_uom_id or rec_id.duration <= 0:
+        for record in self:
+
+            if not record.duration_uom_id or record.duration <= 0:
                 continue
 
-            procurement_ids = \
-                rec_id.dispensed_ids.filtered(
+            procurements = \
+                record.dispensed_ids.filtered(
                     lambda r: r.state == 'done'
                 ).sorted(
                     key=lambda r: r.date_planned
                 )
 
-            if not procurement_ids:
-                rec_id.dispense_remain_qty, rec_id.dispense_remain_day = 0, 0
+            if not procurements:
+                record.dispense_remain_qty, record.dispense_remain_day = 0, 0
                 continue
+            first_proc = procurements[0]
 
-            first_proc = procurement_ids[0]
             date_first_dispense = fields.Datetime.from_string(
                 first_proc.date_planned,
             )
             date_delta = datetime.now() - date_first_dispense
             days_passed = date_delta.days
 
-            if day_uom_id != rec_id.duration_uom_id:
-                day_qty = rec_id.duration_uom_id._compute_quantity(
-                    rec_id.duration, day_uom_id,
+            if day_uom_id != record.duration_uom_id:
+                day_qty = record.duration_uom_id._compute_quantity(
+                    record.duration,
+                    day_uom_id,
                 )
             else:
-                day_qty = rec_id.duration
-            total_qty = rec_id.qty * (rec_id.refill_qty_original + 1.0)
+                day_qty = record.duration
+
+            # @TODO
+            # What should happen if day_qty is 0?
+            if not day_qty:
+                continue
```

Comment:
@lasley I added this code here to prevent the comment below comment from happening.

Just realized this should be checking if day_qty is 0 instead. Is there something else you have in mind if it is 0 instead of continue, or is that good?